policy/daemon_test.go

```diff
@@ -773,6 +778,142 @@ func Test_policyd_CheckPolicy(t *testing.T) {
 	}
 }
 
+func Test_policyd_CheckPolicy_goroutine(t *testing.T) {
```

Comment:
not sure if it is a good practice. as there has so many factories affecting the program runtime (GC time, memory usage, etc) making the test case meaningless... but maybe it is better to have.

Reply:
just to test no leakage...
for TDD, add a test, fix a problem

Reply:
![](https://vignette.wikia.nocookie.net/evchk/images/1/16/Repetitive_error.jpg/revision/latest?cb=20070902105305)